internal/cmd/visibility/delete_visibility.go

```diff
@@ -46,31 +47,24 @@ func (dv *DeleteVisibilityCmd) Validate(args []string) error {
 		return fmt.Errorf("id is required")
 	}
 
-	dv.ids = args
+	dv.id = args[0]
```

Comment:
Check if only one name provided here also

Reply:
There might be additional args (generic params)